docs/notifications.md

```diff
@@ -1,2 +1,106 @@
+
+[![mobile menu notifications link](./img/notifications/mobile-link.png)](img/notifications/mobile-link.png)
+
+### Priority notifications
+
+For critical notifications that require a higher degree of visibility, notifications can be designated as "priority" and will appear more prominently, fixed above the
+application top bar. In the case that the user has multiple unseen priority notifications, a generic message will be displayed, featuring the priority notifications
+count and a link to the notifications page.
+
+[![priority notification](./img/notifications/priority.png)](img/notifications/priority.png)
+
+### Notifications page
+
+On the notifications page, users can view their notifications, dismiss them, and follow calls to action. They can also click the
+'Dismissed' tab to view notifications they've previously dismissed.
+
+Priority notification float to the top of the list and appear with a small icon to the left of the notification's content.
+
+[![notifications page](./img/notifications/notifications-page.png)](img/notifications/notifications-page.png)
+
+## Technical implementation
+
+Notifications depend on a JSON feed containing certain attributes and flags. See the [configuraton]() doc for information about how to
+point uw-frame to your desired feed.
+
+### Example notification
+
+*Note: All notifications must be contained within the `notifications` array.*
+
+```json
+{
+  "notifications": [
+    {
+      "id": 1,
+      "groups" : ["Users - Service Activation Required"],
+      "title"  : "You need to modify your NetID account to activate essential UW Services.",
+      "actionURL" : "https://www.mynetid.wisc.edu/modify",
+      "actionAlt" : "Activate Services",
+      "dismissable" : true,
+      "priority" : false
+    }
+  ]
+}
+```
+
+**Attribute breakdown**
+
+- **id**: A unique number to identify the notification. This is used for sort order on the notifications page.
+- **groups**: An attribute to optionally show notifications only to specific groups (ex. Manifest groups, uPortal groups). **Must contain at least one value**. Using the "Everyone" group will make
+your notification visible to all users. Contact your portal development team for more information about group filtering.
+- **title**: The text to be displayed as the notification's main content. **Be concise!** Try to limit your notification's title to ~140 characters. Longer titles are less likely
+to be read and can cause minor display errors.
+- **actionURL**: A URL where users can get more information or respond to calls to action.
+- **actionAlt** (*optional*): Applies an aria-label to the notification title. Use this if vision-impaired users might need additional context to understand your notification.
+- **dismissable**: Set to true if users should be able to dismiss the notification from their list. This also works for dismissing priority notifications from their fixed position above the top bar.
+**This should almost always be true**.
```

Comment:
This should be true when notifications are not self-dismissing, that is, when users do not drop out of their audience groups when they address whatever they're being notified about.